web/src/compose_fade.ts

```diff
@@ -30,8 +30,6 @@ type MessageContainer = {
     include_sender: boolean;
     is_hidden: boolean;
     mention_classname: string | null;
-    message_edited_notices_alongside_sender: boolean;
-    message_edited_notices_for_status_message: boolean;
```

Comment:
These seem to have been accidentally introduced in 1fb03a27d826f1805c9c854b21a9992bcee55ca9. @LaPulgaaa FYI.

Reply:
That's my bad. Thanks for the fix.